diff -urN phoronix-test-suite/install-sh phoronix-test-suite-patched/install-sh
--- phoronix-test-suite/install-sh	2011-05-18 05:00:59.000000000 +1100
+++ phoronix-test-suite-patched/install-sh	2012-01-27 03:36:52.321366495 +1100
@@ -50,6 +50,7 @@
 mkdir -p $DESTDIR$INSTALL_PREFIX/share/appdata/
 mkdir -p $DESTDIR$INSTALL_PREFIX/share/applications/
 mkdir -p $DESTDIR$INSTALL_PREFIX/share/icons/hicolor/48x48/apps/
+mkdir -p $DESTDIR$INSTALL_PREFIX/share/icons/hicolor/64x64/apps/
 mkdir -p $DESTDIR$INSTALL_PREFIX/share/man/man1/
 mkdir -p $DESTDIR$INSTALL_PREFIX/share/phoronix-test-suite/
 mkdir -p $DESTDIR$INSTALL_PREFIX/share/doc/phoronix-test-suite/
--- phoronix-test-suite/install-sh.orig	2015-07-26 20:33:12.800111473 +0100
+++ phoronix-test-suite/install-sh	2015-07-26 20:33:40.853113578 +0100
@@ -76,8 +76,8 @@
 cp pts-core/static/phoronix-test-suite-launcher.desktop $DESTDIR$INSTALL_PREFIX/share/applications/
 cp pts-core/static/phoronix-test-suite.appdata.xml $DESTDIR$INSTALL_PREFIX/share/appdata/
 
-mkdir -p $DESTDIR$INSTALL_PREFIX/../usr/lib/systemd/system/
-cp deploy/*-systemd/*.service $DESTDIR$INSTALL_PREFIX/../usr/lib/systemd/system/
+mkdir -p $DESTDIR$INSTALL_PREFIX/../lib/systemd/system/
+cp deploy/*-systemd/*.service $DESTDIR$INSTALL_PREFIX/../lib/systemd/system/
 
 # mkdir -p $DESTDIR$INSTALL_PREFIX/../etc/init/
 # cp pts-core/static/upstart/*.conf $DESTDIR$INSTALL_PREFIX/../etc/init/
